tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -477,6 +478,7 @@ handle_build() {
 		install_qemu_snp_experimental
 		install_shimv2
 		install_tdvf
+		install_ovmf
```

Comment:
@fitzthum, we're trying to keep those alphabetically sorted. Mind to move this entry after nydus-tarball and before qemu-tarball ?